charts/featureform/charts/feature-server/templates/deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
             - containerPort: {{ .Values.metrics.port }}
           env:
             - name: SERVING_PORT
-              value: 0.0.0.0:{{ .Values.serving.port }}
+              value: "{{ .Values.serving.port }}"
```

Comment:
why make this a different convention than metadata port?

Reply:
changed to be consistent